app/src/main/res/values/strings.xml

```diff
@@ -1387,8 +1387,9 @@
     <string name="summary_show_home_on_boot">Bring up xDrip screen automatically after device boots</string>
     <string name="title_show_home_on_boot">Show xDrip on Boot</string>
     <string name="eula">EULA</string>
-    <string name="summary_plus_accept_follower_actions">Treatments and Calibrations from followers will be accepted</string>
+    <string name="summary_plus_accept_follower_actions">Treatments, Calibrations and remote sensor stop from followers will be accepted</string>
```

Comment:
I think this might be cleaner as "Treatments, Calibrations and Sensor Stop from followers will be accepted"

Reply:
Thanks.  Updated